packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -168,6 +168,13 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
     },
     [searchParams, pathname, router]
   );
+
+  useEffect(() => {
+    if (calendars.connectedCalendars.length > 0) {
```

Comment:
Isn't the only supported overlay calendar google calendar now? Aka if you install MS outlook and the switch in this case is enabled, would it work?